src/app.js

```diff
@@ -1 +1,29 @@
+/* eslint-disable no-console */
 'use strict';
+
+const operation = process.argv.slice(2)[0];
+
+const numbers = process.argv.slice(3).map(elem => Number(elem));
+
+if (!numbers.length) {
+  console.log(`There is nothing to ${operation}`);
+} else {
+  switch (operation) {
+    case 'add':
+      const resultAdd = numbers
+        .reduce((element, sum) => sum + parseInt(element));
+
+      console.log(resultAdd);
+      break;
+
+    case 'multiply':
+      const resultMultiply = numbers
+        .reduce((element, sum) => sum * Number(element));
+
+      console.log(resultMultiply);
+      break;
+
+    default:
+      console.log('Operation is not supported!');
+  }
```

Comment:
You can declare result with `let` and assign there conditionally. And just use one console.log in the end
```suggestion
} else {
  let result;
  
  switch (operation) {
    case 'add':
      result = numbers.reduce(
        (element, sum) => sum + parseInt(element)
      );
      break;

    case 'multiply':
      result = numbers.reduce(
        (element, sum) => sum * Number(element)
      );
      break;

    default:
      result = 'Operation is not supported!';
  }
```